plugins/course-apps/calculator/Settings.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import AppSettingsModal from 'CourseAuthoring/pages-and-resources/app-settings-modal/AppSettingsModal';
+import messages from './messages';
+
+/**
+ * Settings widget for the "calculator" Course App.
+ * @param {{onClose: () => void}} props 
+ * @returns 
+ */
+const CalculatorSettings = ({ onClose }) => {
+  const intl = useIntl();
+  return (
+    <AppSettingsModal
+      appId="calculator"
+      title={intl.formatMessage(messages.heading)}
+      enableAppHelp={intl.formatMessage(messages.enableCalculatorHelp)}
+      enableAppLabel={intl.formatMessage(messages.enableCalculatorLabel)}
+      learnMoreText={intl.formatMessage(messages.enableCalculatorLink)}
+      onClose={onClose}
+    />
+  );
+};
+
+export default CalculatorSettings;
```

Comment:
Commentary: This file is mostly unchanged (just moved) but since I changed `CalculatorSettings ` to use `useIntl` instead of `injectIntl`, the indentation all changed and git shows it in the diff as a new file.